serializers.go

```diff
@@ -115,3 +119,33 @@ func topic(labels map[string]string) string {
 	}
 	return buf.String()
 }
+
+func filter(name string, labels map[string]string) bool {
+	if len(match) == 0 {
+		return true
+	}
+	mf, ok := match[name]
+	if !ok {
+		return false
+	}
+
+	for _, m := range mf.Metric {
+		if len(m.Label) == 0 {
```

Comment:
There is one corner case:

1. ```
   MATH="['foo', 'bar{x=\"1\"}', 'foo{y=\"2\"}']"
   ```
2. ```
   MATH="['foo{y=\"2\"}', 'bar{x=\"1\"}', 'foo']"
   ```

Depending on the order, `foo{z="3"}` may be accepted (1) or discarded (2).

Reply:
For current implementations, whether `MATH1` or `MATH2`, `foo{z="3"}` will be accepted, regardless of order